### ISSUE TITLE
Fix build with go-pie

### ISSUE THREAD

**Issue by TheEdgeOfRage** [jesseduffield/lazydocker#23]
I'm using the AUR package to build lazydocker, but since I have go-pie installed instead of pure go, the test.sh file fails. This is the output from running it:

```
?   	github.com/jesseduffield/lazydocker/pkg/gui	[no test files]
?   	github.com/jesseduffield/lazydocker/pkg/config	[no test files]
==31172==ERROR: ThreadSanitizer failed to allocate 0x27b9000 (41652224) bytes at address 178baaf8dc000 (errno: 12)
unexpected fault address 0x0
fatal error: fault
unexpected fault address 0x562eab8e69f0
fatal error: fault
unexpected fault address 0x562eac81f10a
fatal error: fault
[signal SIGSEGV: segmentation violation code=0x2 addr=0x562eac81f10a pc=0x562eac81f10a]

goroutine 1 [running, locked to thread]:
runtime: unexpected return pc for runtime.throw called from 0x562eab8e6338
stack: frame={sp:0x7ffe5d64ac40, fp:0x7ffe5d64ac70} stack=[0xc000058000,0xc000058800)

runtime.throw(0x3, 0x3)
	/usr/lib/go/src/runtime/panic.go:617 +0x74 fp=0x7ffe5d64ac70 sp=0x7ffe5d64ac40 pc=0x562eab36ee44
FAIL	github.com/jesseduffield/lazydocker/pkg/commands	0.008s
```

My distro is Arch and I'm running go-pie 1.12.6

**Comment by dawidd6** [comment]
I think that there is nothing we can do here about it, as it seems to be the problem in Go itself:
https://github.com/golang/go/issues/31384

**Comment by mikhailswift** [comment]
@TheEdgeOfRage 

For the time being I've removed the check step of the AUR package that runs the unit tests.  Until there's resolution on the go issue I don't see a way to add them back.  I think it's reasonable to assume the unit tests are run before a new release is published, so we should be safe to remove them from the AUR package for now.

I just tested and confirmed that this package will now install on an Arch installation with go-pie installed.
